docs/howto/advanced/retry.md

```diff
@@ -64,22 +64,49 @@ between retries:
 ## Implementing your own strategy
 
 - If you want to go for a fully fledged custom retry strategy, you can implement your
-  own retry strategy (though we recommend always keeping a max_retry):
+  own retry strategy by returning a `RetryDecision` object from the
+  `get_retry_decision` method. This also allows to (optionally) change the priority,
+  the queue or the lock of the job. The time to wait between retries can be specified
+  with `retry_in` or alternatively with `retry_at`. If `None` is returned
```

Comment:
Let's mention that they have se same values as `task.configure(schedule_in=..., schedule_at=...)`.

Reply:
(also, let's remove the unordered list. Those can be 2 paragraphs.